jwst/datamodels/schemas/miri_mrsxartcorr.schema.yaml

```diff
@@ -0,0 +1,63 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/miri_mrsxartcorr.schema"
+title: MIRI MRS Cross Artifact Corrections
+allOf:
+- $ref: referencefile.schema
```

Comment:
Will need the EXP_TYPE and P_EXP_TY keywords in the schema. These can be inserted using
```
- $ref: keyword_exptype.schema
- $ref: keyword_pexptype.schema
```

Reply:
Added.